django/contrib/admin/templates/admin/base.html

```diff
@@ -90,7 +90,7 @@
           {% endif %}
         {% endblock messages %}
         <!-- Content -->
-        <div id="content" class="{% block coltype %}colM{% endblock %}">
+        <div id="content" class="{% block coltype %}colM{% endblock %}" tabindex="-1">
```

Comment:
Is this the right element to take users of skip links to? I would have expected it’d be better to take people to the `<div class="content">` element on line 84, so the messages aren’t skipped. 

It’s a bit odd that there is a `<div id="content">` and a separate `<div class="content">`, not too sure what to think of that. It would be nice that the thing we point people to with the skip link has an id that’s (somewhat) user-friendly, as it will show in the URL. It’s annoying that both `content` and `main` are already taken.

Reply:
Totally agree!!

How we could handle this? Using a different/new `id` for the skip link at line `84`?

Also there are styles for both elements:`#content` and `.content`, should this go in a different PR?

Thanks!

Reply:
> Using a different/new id for the skip link at line 84?

Can we do that? 

(I think changing IDs would be best handled separately — if it's needed — because we'd need to call it out in release notes and such... — and I'd rather not get **this** stuck on **that**.)

Reply:
Done!

If it is OK and we get the 👍  from thibaudcolas, you can run the selenium tests, because I've bothered buildbot 🤖 a lot 😆